### PR TITLE
VACMS-17861: Updates VAMC System Billing and Insurance CT to phone field to paragraph field.

### PR DESCRIPTION
**Pull request by dsasser**
## Description

Relates to #17861

## Testing done
Manual.

## Screenshots
Screenshot 2024-09-04 at 2 16 29 PM

## QA steps
- [ ] Login to [Tugboat CMS backend](https://pr19039-zvl6wgjmluylqwdznntnjy5wqnvnmgbr.ci.cms.va.gov/)
- [ ] Go to admin/content/audit/[Phone Numbers](https://pr19039-zvl6wgjmluylqwdznntnjy5wqnvnmgbr.ci.cms.va.gov/admin/content/audit/phone-numbers)
- [ ] Filter content type to VAMC System Billing and Insurance
- [ ] Confirm the Direct Line matches Staff Profile Phone

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.
- [ ] If there are field changes, front end output has been thoroughly checked.

### Select Team for PR review

- [ ] `CMS Team`
- [ ] `Public websites`
- [x] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

### CODE REVIEW

**Comment by jilladams** [comment]
(@jv-agile6 randomly passing through and noticed a flag that there are some conflicts in composer.lock to be resolved, in case Github doesn't notify on that)

File: .ddev/nginx_full/nginx-site.conf
Comment (jv-agile6) [inline .ddev/nginx_full/nginx-site.conf:1]: New nginx config added for updated batch function

File: .github/CODEOWNERS
Comment (jv-agile6) [inline .github/CODEOWNERS:1]: updated docs

File: composer.json
Comment (jv-agile6) [inline composer.json:1]: New libraries added - giggsey/libphonenumber-for-php": "^8.13", and "drupal/codit_batch_operations": "^1.0",

File: config/sync/codit_batch_operations.settings.yml
Comment (jv-agile6) [inline config/sync/codit_batch_operations.settings.yml:1]: batch config

File: config/sync/core.entity_form_display.node.person_profile.default.yml
Comment (jv-agile6) [inline config/sync/core.entity_form_display.node.person_profile.default.yml:1]: updated yml

File: config/sync/core.entity_form_display.node.person_profile.inline_entity_form.yml
Comment (jv-agile6) [inline config/sync/core.entity_form_display.node.person_profile.inline_entity_form.yml:1]: updated yml

File: config/sync/core.entity_view_display.node.person_profile.default.yml
Comment (jv-agile6) [inline config/sync/core.entity_view_display.node.person_profile.default.yml:1]: updated yml

File: config/sync/core.entity_view_display.node.person_profile.teaser.yml
Comment (jv-agile6) [inline config/sync/core.entity_view_display.node.person_profile.teaser.yml:1]: updated yml

File: config/sync/core.entity_view_display.node.vamc_system_billing_insurance.teaser.yml
Comment (jv-agile6) [inline config/sync/core.entity_view_display.node.vamc_system_billing_insurance.teaser.yml:1]: updated yml

File: config/sync/core.extension.yml
Comment (jv-agile6) [inline config/sync/core.extension.yml:1]: batch operations added

File: config/sync/field.field.node.person_profile.field_telephone.yml
Comment (jv-agile6) [inline config/sync/field.field.node.person_profile.field_telephone.yml:1]: updated yml

File: config/sync/views.view.rich_text_field_audit.yml
Comment (jv-agile6) [inline config/sync/views.view.rich_text_field_audit.yml:1]: updated yml audit fields

File: docroot/modules/custom/va_gov_batch/src/cbo_scripts/MigratePhoneFieldToParagraph.php
Comment (jv-agile6) [inline docroot/modules/custom/va_gov_batch/src/cbo_scripts/MigratePhoneFieldToParagraph.php:1]: migration logic

File: docroot/modules/custom/va_gov_vamc/src/EventSubscriber/VAMCEntityEventSubscriber.php
Comment (jv-agile6) [inline docroot/modules/custom/va_gov_vamc/src/EventSubscriber/VAMCEntityEventSubscriber.php:1]: Form alter logic

File: docroot/modules/custom/va_gov_vamc/va_gov_vamc.deploy.php
Comment (jv-agile6) [inline docroot/modules/custom/va_gov_vamc/va_gov_vamc.deploy.php:1]: Deploy build

File: tests/phpunit/va_gov_batch/unit/Class/MigratePhoneFieldToParagraphTest.php
Comment (jv-agile6) [inline tests/phpunit/va_gov_batch/unit/Class/MigratePhoneFieldToParagraphTest.php:1]: Unit test case

**Review by jv-agile6 — APPROVED** [review]
Reviewed code, logic and form and found no defects. 🟢

**Comment by jv-agile6** [comment]
Borked branch 👎

**Pull request closed without merging**